lib/configs/base.js

```diff
@@ -3,16 +3,14 @@
  * This file has been automatically generated,
  * in order to update its content execute "npm run update"
  */
+const globals = require('globals')
 module.exports = {
   parser: require.resolve('vue-eslint-parser'),
   parserOptions: {
-    ecmaVersion: 2020,
+    ecmaVersion: 'latest',
     sourceType: 'module'
   },
-  env: {
-    browser: true,
-    es6: true
-  },
+  globals: globals.browser,
```

Comment:
Do we really need to specify globals here? Or should we remove that and rather instruct users what to do in case some global is missing?

Reply:
I think it would make sense to remove `globals.browser` from the config provided by the plugin. In SSR those global objects don't exist.
We'd also need to change the Flat confign, so that might be better done in a separate PR. What do you think?

Reply:
Yeah, let's do it separately :+1:

Reply:
I've added a bullet point in #2166.